docs/examples/http_exceptions.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from nameko.web.handlers import HttpRequestHandler
+from werkzeug.wrappers import Response
+
+
+class ApplicationError(Exception):
+    pass
+
+
+class HttpError(Exception):
+
+    def __init__(self, payload, status_code=None):
```

Comment:
it seems slightly strange to me to have the exception class (and by extension, the service code) care about things like http status codes. i would expect that decision to be made in `response_from_exception`

`HttpError`, again seems to focus on mechanism rather than (as common with exception classes) _what_ went wrong.

maybe something like `InvalidAgruments` (not a great name, but along those lines)